spec/lib/local-links-manager/import/links_importer_spec.rb

```diff
@@ -104,6 +104,42 @@
 
         expect(link.reload.url).to eq 'http://www.example.com/this-is-now-different'
       end
+
+      context 'when a link is missing from the CSV' do
+        it 'is removed from the database' do
+          service = FactoryGirl.create(:service, lgsl_code: 123, label: 'Service 123')
```

Comment:
The setup for creating a Link is quite involved. Wondering whether it would be worth it to make our factories do more of this work, so that we could just call something like this (without specifying the dependencies):
```
first_link = FactoryGirl.create(:link, url: 'http://example.com/first-link')
second_link = FactoryGirl.create(:link, url: 'http://example.com/second-link')
```

When creating more than one Link (as in this test), we'd have to set up [sequences](https://github.com/thoughtbot/factory_girl/blob/master/GETTING_STARTED.md#sequences) for our `LocalAuthority`, `Service` and `Interaction` factories due to their uniqueness validations on various fields. 

Anyone else have any opinions on this?